src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -1537,7 +1537,7 @@ void SolverInterfaceImpl:: handleExports()
           suffix << _accessorName << ".dt" << _couplingScheme->getTimesteps()-1;
           exportMesh(suffix.str());
           if (context.triggerSolverPlot){
```

Comment:
We can also think about entirely removing this block:

```
if ( context.triggerSolverPlot ) {
...
}
```

**Don't forget to check whether `context.triggerSolverPlot` and corresponding API can be removed**